tap_kafka/local_store.py

```diff
@@ -137,7 +137,13 @@ def insert(self, message: str) -> None:
         """Insert a new message with a generated timestamp
         Returns the last timestamp of the last inserted message"""
         # Add to the in-memory batch to avoid too frequent I/O write
-        self.messages_to_persist.append(message)
+        message_to_append = message
+        if isinstance(message, bytes):
+            message_to_append = message.decode('utf-8')
+        else:
+            message_to_append = message
```

Comment:
this is redundant cuz `message_to_append` is already initialized to `message` on line 140

Reply:
thank you, fixed by 88ee091e20d19008ec7484a9fda0c56e4f2ab2be